pkg/test/useraccount/assertion.go

```diff
@@ -0,0 +1,66 @@
+package uatest
```

Comment:
same question about this pkg name

Reply:
and same answer :-)

Reply:
please see https://github.com/codeready-toolchain/toolchain-common/pull/34#discussion_r319404170 about the package name